src/corehost/cli/context_contract.h

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: maybe call the file `corehost_context_contract` or `hostpolicy_context_contract`... 
Since it lives in the shared directory it's rather confusing where it belongs or what it is.